### PR TITLE
Fix #97: Display only the rewards summary when viewing local URLs

### LINKED ISSUE #97: Localhost url should hide publisher info
With url provided: `"http://localhost:6571/about/home/#panel=0"`

actual:

Zrzut ekranu 2019-06-17 o 23 50 31

expected(summary only)
Zrzut ekranu 2019-06-18 o 00 01 27

### PR DESCRIPTION
**Pull request by kylehickinson**

### CODE REVIEW

File: BraveRewardsUI/Wallet/WalletViewController.swift [hunk 1]
```diff
@@ -125,7 +128,11 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
     }
     
     if state.ledger.isEnabled {
-      height = RewardsUX.preferredPanelSize.height
+      if isLocal && rewardsSummaryView.rows.isEmpty {
+        height = 400 // TODO: Generate this number somehow
```
[the remark this answers is not in the document]
Comment (kylehickinson) [reply BraveRewardsUI/Wallet/WalletViewController.swift]: Where does the 20 come from?

[the remark this answers is not in the document]
Comment (danishjafri88) [reply BraveRewardsUI/Wallet/WalletViewController.swift]: Its the padding b/w Header and SummaryLayoutGuide:
`summaryLayoutGuide.snp.makeConstraints {
        $0.top.equalTo(self.headerView.snp.bottom).offset(20.0)`

**Comment by iccub** [comment]
Can you show a screenshot of the screen with localhost url now?

**Comment by danishjafri88** [comment]
> Can you show a screenshot of the screen with localhost url now?

Simulator Screen Shot - iPhone X - 2019-07-02 at 23 14 41

**Comment by kylehickinson** [comment]
| Non-Empty | Empty |
| -- | -- |
| Simulator Screen Shot - iPhone 8 - 2019-07-02 at 13 44 46 | Simulator Screen Shot - iPhone 8 - 2019-07-02 at 13 44 41 |

**Pull request merged**